gk/main.c

```diff
@@ -411,16 +327,22 @@ gk_process_granted(struct flow_entry *fe, struct ipacket *packet)
 
 	fe->u.granted.budget_byte -= pkt->data_len;
 	renew_cap = now >= fe->u.granted.send_next_renewal_at;
-	if (renew_cap)
+	if (renew_cap) {
 		fe->u.granted.send_next_renewal_at = now +
 			fe->u.granted.renewal_step_cycle;
+		priority = 2;
```

Comment:
I think we should avoid having 2 as a magic number, and instead add a constant like `PRIORITY_RENEW_CAP` for better readability. Same thing with priority 1 near the top of this function.